Add --internal flag for export kubecfg that targets the internal dns name (#9732)
kubernetes/kops: docs/cluster_spec.md

      idleTimeoutSeconds: 300
```

You can use a valid SSL Certificate for your API Server Load Balancer. Currently, only AWS is supported.

> Note that when using `sslCertificate`, client certificate authentication will no longer work through the load balancer, for example credentials generated via `kops export kubecfg`. This can be bypassed with the `--internal` flag which will target the master instances directly rather than the load balancer. Security groups may need to be opened to allow access from the clients to the master instances' port TCP/443, for example by using the `additionalSecurityGroups` field on the master instance groups.

Reviewer: ```suggestion
Note that when using `sslCertificate`, client certificate authentication, such as with the credentials generated via `kops export kubecfg`, will not work through the load balancer. A `kubecfg` that bypasses the load balancer may be created with the `--internal` flag to `kops update cluster` or `kops export kubecfg`. Security groups may need to be opened to allow access from the clients to the master instances' port TCP/443, for example by using the `additionalSecurityGroups` field on the master instance groups.
```
